### PR TITLE
Fix notification strings (#4004)

### PR DESCRIPTION
**Pull request by char32**
This patch fixes the notification strings in the
case that neither name nor message should be shown.

This changes the string names and values for all
language and in the respective source file.

Additionally the german translation is adjusted too.

Fixes issue #4004

### CODE REVIEW

**Comment by haffenloher** [comment]
Please don't change the translations, `res/values/strings.xml` is enough :-)
The other XML files are generated through [Transifex](https://www.transifex.com/moxie/textsecure-official/).

**Comment by char32** [comment]
@haffenloher Thanks for the Review! :-)

Ok I will remove the Translation today in the eveing. But the Change of the string_name itself in all language files (to SingleRecipientNotificationBuilder_new_message) should remain, right? So it is enough to remove the Change of the german Translation value, right?

**Comment by ChaoticMind** [comment]
@char32 no, the localized files are generated whole when exporting the translations from Transifex, including the string names. You only need to change `res/values/strings.xml`.

**Comment by char32** [comment]
@ChaoticMind Thanks! That makes it a lot easier. Didn't know about TransiFlex before. But yeah I am still learning how everything in TextSecure works together :-)
I will Change it later today then and update this  PR.

**Comment by char32** [comment]
Ok, the proposed changes are done :-)

File: src/org/thoughtcrime/securesms/notifications/SingleRecipientNotificationBuilder.java [hunk 1]
```diff
@@ -131,9 +131,9 @@ public void setTicker(@NonNull Recipient recipient, @Nullable CharSequence messa
     if (privacy.isDisplayMessage()) {
       setTicker(getStyledMessage(recipient, message));
     } else if (privacy.isDisplayContact()) {
-      setTicker(getStyledMessage(recipient, context.getString(R.string.SingleRecipientNotificationBuilder_new_signal_message)));
+      setTicker(getStyledMessage(recipient, context.getString(R.string.SingleRecipientNotificationBuilder_signal)));
     } else {
-      setTicker(context.getString(R.string.SingleRecipientNotificationBuilder_new_signal_message));
+      setTicker(context.getString(R.string.SingleRecipientNotificationBuilder_signal));
```
Comment (moxie0) [inline src/org/thoughtcrime/securesms/notifications/SingleRecipientNotificationBuilder.java]: shouldn't these be "new message"?

Comment (char32) [inline src/org/thoughtcrime/securesms/notifications/SingleRecipientNotificationBuilder.java]: @moxie0 
I am not sure. According to #4004 every occourance of
"New TextSecure Message" should be replaced with "TextSecure". Ok now with "Signal".

Should I change it to SingleRecipientNotificationBuilder_new_message ? From the context I would would preffer SingleRecipientNotificationBuilder_new_message, too.

**Comment by char32** [comment]
Ok, I looked over it again and changed it accordingly. The two occourances should be new_message , like you stated, for sure! :-) Changed that accidentally in the first & second patchset....

**Pull request closed without merging**